modules/media/jsx/mediaIndex.js

```diff
@@ -113,7 +117,7 @@ class MediaIndex extends Component {
     * XXX: Currently, the order of these fields MUST match the order of the
     * queried columns in _setupVariables() in media.class.inc
     */
-    const options = this.state.data.fieldOptions;
+    const options = this.state.fieldOptions;
```

Comment:
Hi @HenriRabalais I know this is just accessing the maxUploadSize but is it good practice to avoid possible state mutation by not grabbing something directly from the state?
Instead doing something like:
`const options = Object.assign({}, this.state.fieldOptions);`

Reply:
What mutation are you referring to? This const seems to only ever be read.

Reply:
@driusan just a concern for a potential problem in the future if new code is introduced. @HenriRabalais told me it's bad to directly grab a value from the state instead of copying it.